paseos/actors/actor_builder.py

```diff
@@ -0,0 +1,134 @@
+from loguru import logger
```

Comment:
Can we have explicit tests for the factory class?

Reply:
Added in https://github.com/aidotse/PASEOS/pull/26/commits/8a80425e748ca9c796cdc4420095df6a97f1f3de